token-lending/program/tests/genesis_accounts.rs

```diff
@@ -12,9 +12,12 @@ use spl_token_lending::{
 };
 
 #[tokio::test]
-async fn test_success() {
+async fn test_genesis_accounts() {
     let (mut test, lending) = setup_test();
 
+    // TODO: Remove this next line to restore 200,000 limit
+    test.set_bpf_compute_max_units(240_000);
```

Comment:
This is a problem, token-lending/program/tests/genesis_accounts.rs requires over 200,000 now